gazebo_plugins/include/gazebo_plugins/gazebo_ros_openni_kinect.h

```diff
@@ -115,7 +115,10 @@ namespace gazebo
     private: sensor_msgs::PointCloud2 point_cloud_msg_;
     private: sensor_msgs::Image depth_image_msg_;
 
+    /// \brief Minimum range of the point cloud
     private: double point_cloud_cutoff_;
```

Comment:
Maybe we can tic-toc these two changes and stay backward compatible?

Otherwise I'll merge this pr as is, and create new pull requests to rename the class member and the sdf parameter later.